wasmtime-wasi/src/host_impls.rs

```diff
@@ -92,6 +92,33 @@ pub fn wasmtime_ssp_proc_exit(rval: wasm32::__wasi_exitcode_t) {
     ::std::process::exit(rval as i32)
 }
 
+pub unsafe fn wasmtime_ssp_fd_prestat_get(
+    prestats: *mut host::fd_prestats,
```

Comment:
Can this be an `&mut` instead of `*mut` here?

Reply:
I've changed it `&mut` and made the fn safe 😎